src/main/java/io/github/ardoco/textproviderjson/converter/JsonConverter.java

```diff
@@ -19,7 +19,7 @@
  **/
 public final class JsonConverter {
 
-    private static final String SCHEMA_PATH = "schemas/text.json";
+    private static final String SCHEMA_PATH = "src/main/resources/schemas/text.json";
```

Comment:
You should use `this.getClass().getResource()` or `getResourceAsStream()`. Otherwise, it might not work in compiled jar

Reply:
simply replace FileInputStream by `this.getClass().getResourceAsStream()`